pwa/src/components/attributes/attribuutForm.tsx

```diff
@@ -21,13 +21,21 @@ import {isLoggedIn} from "../../services/auth";
 import FlashMessage from 'react-flash-message';
 import {navigate} from "gatsby-link";
 import ElementCreationNew from "../common/elementCreationNew";
+import ApplicationForm from "../applications/applicationForm";
 
-export default function AttributeForm({id, entity}) {
-  const [context, setContext] = React.useState(null);
+
+interface AttributeFormProps {
+  id: string
+  entity
```

Comment:
`entity` mist een type (in dit geval is `entity` een string dus: `entity: string`.

Ook mag een object comma separated worden dus:

```
id: string,
entity: string,
```